cc/dual_net/tpu_dual_net.cc

```diff
@@ -158,11 +165,12 @@ TpuDualNet::TpuDualNet(const std::string& graph_path,
 
   // Run warm-up inferences on all sessions.
   // Tensorflow lazily initializes the first time Session::Run is called,
-  // which can take hundreds of milliseconds. This intefers with time control,
+  // which can take hundreds of milliseconds. This interfers with time control,
   // so explicitly run inference once during construction.
+  // TODO(tommadams): Is this still necessary without 'time control'?
```

Comment:
I'm not sure what you mean by this comment. Time control is a feature of the [MctsPlayer](https://github.com/tensorflow/minigo/blob/ae1eec2e40fc4d73616bf000192a1255d8f3c793/cc/mcts_player.cc#L135) and has nothing to do with DualNet.

Reply:
Aha, I put 'time control' in quotes because I didn't really understand what it refers to. Now it makes sense, thanks. Removed.